frontend/ucc_gui/src/containers/Projects/ActiveProject/Step_3/LearnNewSkill/LearnNewSkill.js

```diff
@@ -33,28 +33,39 @@ class LearnNewSkill extends React.Component {
             projectMission: '',
             //project_join_date: '',
             challengeStatus: '',
-            projectCategory: ''
+            projectCategory: '',
+            challenge_present: false
         }
     };
 
     componentDidMount() {
-        AxiosConfig.get(`charityproject/activeProjectList/${cookie.load('user_email')}`)
-            .then(res => {
-                for (let i = 0; i < res.data.active_project_list.length; i++) {
-                    if (res.data.active_project_list[i].project_id === parseInt(this.state.projectId)) {
-                        console.log("inside");
+        Promise.all([
+            AxiosConfig.get(`charityproject/activeProjectList/${cookie.load('user_email')}`),
+            AxiosConfig.get(`charityproject/LearnNewSkill/${this.state.projectId}/${cookie.load('user_email')}`)])
+            .then(([res1, res2]) => {
```

Comment:
what is res1, res2. Like are you getting data from API in array format?

Reply:
Its response from 2 get calls.